tests/e2e_runner.js

```diff
@@ -102,6 +107,10 @@ testsToRun.reduce((previous, relativeName) => {
     currentFileName = relativeName.replace(/\.ts$/, '');
     const start = +new Date();
 
+    if(argv.skip && skips.some(skip => currentFileName.includes(skip))) {
```

Comment:
To accelerate the testing. Sometimes you may want to run tests without reruning some steps of the setup, especially when you change something minor only. It's only helpful in development.